eZ/Publish/API/Repository/Tests/SearchEngineIndexingTest.php

```diff
@@ -848,6 +848,33 @@ public function getSpecialFullTextCases()
     }
 
     /**
+     * Test FullText search on user first name and last name.
+     *
+     * @see https://jira.ez.no/browse/EZP-27250
+     */
+    public function testUserFullTextSearch()
+    {
+        $repository = $this->getRepository();
+        $searchService = $repository->getSearchService();
+
+        $criterion = new Criterion\LogicalAnd(
+            [
+                new Criterion\FullText('Jon Snow'),
+                new Criterion\ContentTypeIdentifier('user'),
+            ]
+        );
+        $query = new Query(['filter' => $criterion]);
+        $results = $searchService->findContent($query);
+        // sanity check
+        self::assertEquals(0, $results->totalCount);
+
+        $this->createUser('TestUser', 'Jon', 'Snow');
+        $this->refreshSearch($repository);
+        $results = $searchService->findContent($query);
+        self::assertEquals(1, $results->totalCount);
```

Comment:
In 8fce947, I reorganized code in this test so sanity check is not needed and as you want I'm checking this by id :) (actually I still need to check `totalCount` to avoid undefined index notice in `searchHits` later on).

Also improved there readability of variable embedded in string, as discussed.